Private/Get/Get-BTWildcardRecord.ps1

```diff
@@ -16,7 +16,17 @@ function Get-BTWildcardRecord {
         'DnsKey','DS','NSec','NSec3','NSec3Param','Tlsa')
         $WildcardExists = $false
         foreach ($rrtype in $RRTypes) {
-            if (Get-DnsServerResourceRecord -ComputerName $domain -ZoneName $domain -RRType $rrtype -Name '*' -ErrorAction Ignore) {
+            # Find a valid DC from $domain and get it's resource record
```

Comment:
This 👍 code to find a valid DC is included in three functions. It might be worth writing once for the module, with a variable that can be called from any function (like DNSServers).